src/microbe_stage/editor/PatchMapEventIcon.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Godot;
+
+/// <summary>
+///   Texture for patch event
+/// </summary>
+public partial class PatchMapEventIcon : TextureRect
```

Comment:
Why is there a blank script here? This will unnecessarily eat up performance...

Reply:
Itsn't needed here: `var eventIcon = eventIconScene.Instantiate<PatchMapEventIcon>();`?